gobblin-utility/src/main/java/org/apache/gobblin/util/retry/RetryerFactory.java

```diff
@@ -127,7 +124,7 @@ public boolean apply(Throwable t) {
     return RetryerBuilder.<T> newBuilder()
         .retryIfException(RETRY_EXCEPTION_PREDICATE)
         .withWaitStrategy(WaitStrategies.exponentialWait(config.getLong(RETRY_MULTIPLIER),
-                                                         config.getLong(RETRY_INTERVAL_MS),
+                                                         config.getLong(RETRY_MAX_INTERVAL_MS),
```

Comment:
doesn't this break backward compatibility?

first off, I prefer your new name to the existing one... but existing code out there is setting `RETRY_MULTIPLIER` and that no longer means anything for EB--which is the default strategy.  I'd say it's too late to change naming.

less severely, but since this is a general library, is it possible certain existing clients aren't on such a large time-scale and would instead consider seconds (rather than ms) way too long?

...or do you think I'm being too conservative?